### DISCUSSION TITLE
初入機械鍵盤分享

### DISCUSSION THREAD

**Discussion by yucheng1028** [1101-GS4719-C/Python-Community-GS4719-C#507] [閒聊八卦]
### 初入機械鍵盤分享

古人曾說過 :「工欲善其事，必先利其器。」 為了學好程式碼，你值得一把適合你的機械鍵盤。

我從第一把 G512 紅軸 換到 GPRO 茶軸 換到 ROG 青軸 

終於 找到了最適合自己的一把
ROG 月光白 紅軸 
但我最近又手癢想換客製化的了...

<img src="https://user-images.githubusercontent.com/81725559/159636536-4b3bcc82-a34e-4e13-82a0-986a4ef541ed.jpg" height="300" alt="微信小程式"/><br/>

歡迎大家入坑機械鍵盤

_大眾品牌_ ( 有保固有保障 ) : 
https://agirls.aotter.net/post/60380

_客製化入門_ ( 1000~2000 ) : 
(腹靈F12)
https://www.ptt.cc/bbs/Key_Mou_Pad/M.1604921879.A.EF1.html
(CIY 68)
https://www.ptt.cc/bbs/Key_Mou_Pad/M.1642593044.A.4ED.html

_客製化進階_ ( 這水很深 慎入 ) :
https://www.inpad.com.tw/product.php?act=list&cid=984

**Comment by d26488399** [comment]
謝謝大神整理，讓我有購買方向。

**Reply by d26488399** [reply]
我這裡也有一把很推薦的
https://www.youtube.com/watch?v=dQw4w9WgXcQ

**Reply by yucheng1028** [reply]
我也很推這把
他介紹很詳細

**Comment by yi-1111** [comment]
推推，最近也想入坑，感謝經驗分享

**Comment by chupochun** [comment]
謝謝大神分享，我最近也好想買一個鍵盤

**Comment by 11timyyy** [comment]
個人是比較喜歡茶軸~

**Comment by Sazmafuji** [comment]
我現在也在摸索哪一種鍵盤適合我，我會好好利用你分享的網站去分析分析一下

**Comment by JSHT** [comment]
非客製化機械鍵盤，不打遊戲，後面大多都會上 Filco 或 Leopold 的樣子
另外個人覺得客製化可以選 Gateron 黃軸，潤過手感跟聲音都不錯。

**Reply by yucheng1028** [reply]
大 L 和 大 F 真的都很不錯，目前手邊有ttc金粉軸，很輕反應也快，找個時間來潤一下

**Reply by JSHT** [reply]
另外也想推鍵盤型 Youtube 頻道: [:3ildcat](https://www.youtube.com/c/3ildcat)，主要分享組鍵盤的過程、鍵盤打字對比、打字 ASMR，整體節奏跟組裝過程的氛圍都很讚。

**Comment by ThomasMeng0515** [comment]
想請問大家推薦Razer的機械鍵盤嗎，最近想入坑

**Comment by hapiwangy** [comment]
一個好的鍵盤真的對需要長時間進行鍵盤輸入的使用者來說真的很重要。除了舒適感之外，好的設計對於身體的危害也比較少，因此慎選鍵盤真的是很重要的

**Comment by leon10456** [comment]
個人使用銀軸 反應速度跟手感真的沒話說 但是真的貴
貴不是他的缺點 是我的....

**Comment by etin2468** [comment]
最近想要組電腦，完全從零開始，所以連周邊的產品也在到處看看，感謝樓上大神們的分享

**Reply by Ben6uo** [reply]
要記得SSD插法跟HDD插法一樣 記得要很大力插進去喔
RAM的話要拿槌子打下去

**Comment by PoooYen** [comment]
這看起來是很深的一個坑.........感謝分享

**Comment by xkllkx** [comment]
謝謝推坑，但是好鍵盤真的要價不斐....

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，我會好好分析哪種鍵盤適合我。

**Comment by SamTU0222** [comment]
謝謝分享，最近也想購入機械式鍵盤

**Comment by Yuhsin000** [comment]
顏色很賞心悅目ㄟ，使用時心情會很好

**Comment by Ben6uo** [comment]
有錢人阿 還有RGB 不過希望可以小聲憶點

**Comment by jason79461385** [comment]
![image](https://user-images.githubusercontent.com/75102829/163746547-8b8c6d9f-bf1e-472e-8f49-260febd6f0fe.jpeg)

靜電容也是不錯喔可以考慮看看
各人推薦55g+原廠靜音化
或是bke domes light

**Comment by lx3211** [comment]
機械鍵盤打起來手感真的好
一開始用筆電打字不太習慣
太扁了

**Comment by cordydeng** [comment]
在網路上看到好多不同的鍵盤 , 有的鍵帽還有圖案 , 感覺會是一個很大的坑

**Comment by sys511613** [comment]
之前買了大L的茶軸
最近有點手癢想買個靜電容鍵盤來玩玩，但是好貴QQ

**Comment by awkward-willy** [comment]
前陣子也買了把茶軸來玩
不過住宿舍的我小擔心會不會吵到室友
好在目前看來他們沒有怨言啦
不過機械式鍵盤真的是大坑 沒事還是別亂跳好

**Comment by stan890425** [comment]
敲起來就是不一樣啦

**Comment by alexthebigpp** [comment]
機械鍵盤有夠香，現在再用ducky的茶軸打起來是真的爽

**Comment by bloodtrail0511** [comment]
可是我沒錢QQ
